charts/snapshot-controller/templates/deployment_controller.yaml

```
@@ -70,4 +70,14 @@ spec:
       {{- if .Values.controller.priorityClassName }}
       priorityClassName: {{ .Values.controller.priorityClassName }}
       {{- end }}
+      {{- if kindIs "bool" .Values.controller.hostNetwork }}
+      hostNetwork: {{ .Values.controller.hostNetwork }}
+      {{- end }}
+      {{- with .Values.controller.dnsConfig }}
+      dnsConfig:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.controller.dnsPolicy }}
+      dnsPolicy: {{ . }}
+      {{- end }}
```

I believe this can be simplified to just:
```suggestion
      hostNetwork: {{ .Values.controller.hostNetwork }}
      dnsConfig:
        {{- toYaml .Values.controller.dnsConfig | nindent 8 }}
      dnsPolicy: {{ .Values.controller.dnsPolicy }}
```